src/index.ios.js

```diff
@@ -0,0 +1,2 @@
+import DrawerLayout from 'react-native-drawer-layout';
```

Comment:
What happens? This is a valid export syntax in ES2015 and works as expected for me with just `es2015` preset.